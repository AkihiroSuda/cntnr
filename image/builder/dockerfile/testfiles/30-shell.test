FROM alpine:3.7
SHELL ["/bin/bash", "-c"]

RUN ["/install.sh", "-a"]
ENTRYPOINT ["/entrypoint.sh", "-x"]
CMD ["/entrypoint.sh", "-x"]

RUN /install.sh -a
ENTRYPOINT /entrypoint.sh -x
CMD /entrypoint.sh -x

SHELL ["/bin/sh", "-c"]

RUN /install.sh -a
ENTRYPOINT /entrypoint.sh -x
CMD /entrypoint.sh -x