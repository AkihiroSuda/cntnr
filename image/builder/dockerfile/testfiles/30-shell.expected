FROM alpine:3.7
RUN "/install.sh" "-a"
ENTRYPOINT "/entrypoint.sh" "-x"
CMD "/entrypoint.sh" "-x"
RUN "/bin/bash" "-c" "/install.sh -a"
ENTRYPOINT "/bin/bash" "-c" "/entrypoint.sh -x"
CMD "/bin/bash" "-c" "/entrypoint.sh -x"
RUN "/bin/sh" "-c" "/install.sh -a"
ENTRYPOINT "/bin/sh" "-c" "/entrypoint.sh -x"
CMD "/bin/sh" "-c" "/entrypoint.sh -x"